jobs/JEVS_NFCENS_WAVE_GRID2OBS_STATS

```diff
@@ -58,7 +58,7 @@ export VERIF_CASE=${VERIF_CASE:-grid2obs}
 ####################################
 export COMIN=${COMIN:-$(compath.py ${envir}/com/${NET}/${evs_ver})}
 export COMINmodel=${COMINmodel:-$(compath.py ${envir}/com/${RUN}_${MODELNAME}/${nfcens_ver})}
```

Comment:
Same thing, let's use the specific model name. Change COMINmodel to COMINnfcens.

Reply:
@ShelleyMelchior-NOAA 
I applied the changes.

Reply:
> @ShelleyMelchior-NOAA I applied the changes.

Great -- can you push them? I don't seen them in the PR yet.